ui.apps/src/main/content/jcr_root/apps/venia/components/structure/header/v1/header/header.html

```diff
@@ -32,6 +32,7 @@
                         d="M1 1h4l2.68 13.39a2 2 0 0 0 2 1.61h9.72a2 2 0 0 0 2-1.61L23 6H6"></path></svg>
                 </span>
                 </button>
+                <sly data-sly-resource="${resource.path}/minicart"/>
```

Comment:
I would suggest to add a test as well here which checks if the minicart child exists because currently AEM almost  gets frozen if minicart is missing. (See the inclusion of navigation above.)